pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -287,13 +287,21 @@ def _dataset_iterator(g, prefix=''):
 # =============================================================================
 # Functions to load triggers
 # =============================================================================
-def load_triggers(input_file, ifos, vetoes, rw_snr_threshold=None,
-                  slide_id=None):
+def _load_triggers(input_file, ifos, data_tag=None, rw_snr_threshold=None,
+                   slide_id=None):
     """Loads triggers from PyGRB output file, returning a dictionary"""
 
+    logging.info("Loading triggers.")
```

Comment:
Kind of a silly point, but looking a few lines below, I wonder if this should say "Loading injections" if `data_tag` is "injs"?